### PR TITLE
129342

### PR DESCRIPTION
**Pull request by silvam11**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]
- [ ] This is a bug
- [x] This is a feature request
- [ ] This is a syntax/style fix
- [ ] This is an improvement to code quality

---

Provide a clear description of changes:
 
[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/AmqpNodeService.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public IdracInfo idracNetworkSettings(IdracNetworkSettingsRequest idracNetworkSe
 
         try {
             IdracNetworkSettingsRequestMessage idracNetworkSettingsRequestMessage =new IdracNetworkSettingsRequestMessage();
-            com.dell.cpsd.rackhd.adapter.rabbitmq.MessageProperties messageProperties = new com.dell.cpsd.rackhd.adapter.rabbitmq.MessageProperties();
+            com.dell.cpsd.rackhd.adapter.model.idrac.MessageProperties messageProperties = new com.dell.cpsd.rackhd.adapter.model.idrac.MessageProperties();
```
Comment (madenb) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/AmqpNodeService.java]: i think we should relook at how this generated class is created and keep it in the same package as the other classes from this api.

File: dne-paqx/pom.xml [hunk 2]
```diff
@@ -236,6 +236,25 @@
...
+                            <removeOldOutput>true</removeOldOutput>
+                            <includeConstructors>true</includeConstructors>
+                            <serializable>false</serializable>
+                            <useCommonsLang3>true</useCommonsLang3>
+                            <includeAdditionalProperties>false</includeAdditionalProperties>
+                            <customAnnotator>com.dell.cpsd.common.contract.enricher.plugins.jsonschema2pojo.MessageAnnotator</customAnnotator>
+                            <customRuleFactory>com.dell.cpsd.common.contract.enricher.plugins.jsonschema2pojo.DefaultRuleFactory</customRuleFactory>
+                        </configuration>
+                    </execution>
+                    <execution>
```
Comment (madenb) [inline dne-paqx/pom.xml]: can we look at keeping the same packaging as the other messages in the same api.

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/AmqpNodeService.java (hunk 1 above)
Comment (silvam11) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/AmqpNodeService.java]: Make sense.

File: dne-paqx/pom.xml (hunk 2 above)
Comment (silvam11) [inline dne-paqx/pom.xml]: Make sense

**Comment by silvam11** [comment]
No, I have not touched the RabbitConfig.java.

**Comment by silvam11** [comment]
Shall I?

**Pull request merged**